UnityClient/Assets/Scripts/LevelLoader.cs

```diff
@@ -5,5 +5,8 @@
 
 public class LevelLoader : MonoBehaviour
 {
-    public void ChangeScene() => SceneManager.LoadScene("Room1");
-}
\ No newline at end of file
+    public void ChangeScene(string loadLevelName)
+    {
+        SceneManager.LoadScene(loadLevelName);
```

Comment:
This can be expression-bodied, though I'll be honest I'm unsure why this method even exists.